CalcualtorLogic.js

```diff
@@ -112,8 +112,8 @@ class Calculator {
         this.resultElement.textContent = this.currentResult;
     }
     onOperatorButtonClick(event) {
-        const curntRsltAndOperclkdBtn = event.target.dataset.operator;
-        this.operator = curntRsltAndOperclkdBtn;
+        const OperclkdBtn = event.target.dataset.operator;
```

Comment:
Please use readable name

Reply:
I did before seeing this comment  but I do not know why it dose not updated in github if so